### DISCUSSION TITLE
Recent completion changes don't allow fuzzy matching

### DISCUSSION THREAD

**Discussion by rodfersou** [ohmyzsh/ohmyzsh#11942] [Help]
For now there is this workaround that seems to work for everybody: https://github.com/robbyrussell/oh-my-zsh/issues/1398#issuecomment-255596509. Have a look and report any issues.

Related: #1398 #5435.
─ @mcornella

---

When I try to change directory:

```
$ cd /s/s.s.l <tab>
```

I expect that it completes as:

```
$ cd /srv/sc.social.like
```

but it is not working.. I  don't know where to check this bug.. it worked fine in my old ubuntu, but this week I start using arch linux, and this bug is driving me crazy..

omz-debug.log
*REDACTED*

**Comment by mcornella** [comment]
This will be the issue to track this problem, for everybody that has it. So please subscribe for updates.

**Comment by rodfersou** [comment]
@mcornella the workaround didn't work for me.. is it possible to explain what commit start the mess? I didn't find it

**Comment by rodfersou** [comment]
nevermind.. I could add the workaround now! thanks

**Comment by jalexvig** [comment]
A workaround from [this post](https://superuser.com/a/815317/686936):

Add the following to your `.zshrc`

```zsh
# 0 -- vanilla completion (abc => abc)
# 1 -- smart case completion (abc => Abc)
# 2 -- word flex completion (abc => A-big-Car)
# 3 -- full flex completion (abc => ABraCadabra)
zstyle ':completion:*' matcher-list '' \
  'm:{a-z\-}={A-Z\_}' \
  'r:[^[:alpha:]]||[[:alpha:]]=** r:|=* m:{a-z\-}={A-Z\_}' \
  'r:|?=** m:{a-z\-}={A-Z\_}'
```
